index.html

```diff
@@ -0,0 +1,278 @@
+                industry's standard</p>
+                <div>
+                  <button type="button" class="button">
+                    <span>Html</span>
+                  </button>
+                  
+                  <button type="button" class="button">
+                    <span class="big-span">Bootstrap</span>
+                  </button>
+                  
+                  <button type="button" class="button">
+                    <span>Ruby</span>
+                  </button>
+  
+                </div>
+                <button type="button" class="project-btn2">See Project</button>
+              </div>
+              <div class="project-section-img1">
+                <h3>Professional Art Printing Data</h3>
+                <p class="paragraph">A daily selection of privately personalized reads no accounts or sign-ups required has been the
+                industry's standard</p>
+                <div>
+                  <button type="button" class="button">
+                    <span>Html</span>
+                  </button>
+                  
+                  <button type="button" class="button">
+                    <span class="big-span">Bootstrap</span>
+                  </button>
+                  
+                  <button type="button" class="button">
+                    <span>Ruby</span>
+                  </button>
+  
+                </div>
+                <button type="button" class="project-btn2">See Project</button>
+              </div>
+              <div class="project-section-img3">
+                <h3>Professional Art Printing Data</h3>
+                <p class="paragraph">A daily selection of privately personalized reads no accounts or sign-ups required has been the
+                industry's standard</p>
+                <div>
+                  <button type="button" class="button">
+                    <span>Html</span>
+                  </button>
+                  
+                  <button type="button" class="button">
+                    <span class="big-span">Bootstrap</span>
+                  </button>
+                  
+                  <button type="button" class="button">
+                    <span>Ruby</span>
+                  </button>
+                </div>
+  
+                <button type="button" class="project-btn2">See Project</button>
+              </div>
+          </div>
+        </section>
+      </section>
+
+      <section id="About" class="about-me">
+        <h2 class="about-me-text">About me</h2>
+        <p class="about-me-paragraph">Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.
+        </p>
+        <button type="button" class="project-btn1 about-btn">Get my Resume</button>
```

Comment:
- [ ] Kindly change the color of the button also.

Reply:
okay done
Thanks!
@vic778